test/core/transport/chttp2/hpack_encoder_test.c

```diff
@@ -103,8 +103,14 @@ static void verify(grpc_exec_ctx *exec_ctx, size_t window_available, int eof,
 
   grpc_transport_one_way_stats stats;
   memset(&stats, 0, sizeof(stats));
-  grpc_chttp2_encode_header(exec_ctx, &g_compressor, 0xdeadbeef, &b, eof, 16384,
-                            &stats, &output);
+  grpc_encode_header_options hopt = {
+      .stream_id = 0xdeadbeef,
+      .is_eof = eof,
+      .use_true_binary_metadata = false,
```

Comment:
Yeah... need to add some... not sure I'll get cycles today (but will try)... they take some time to craft.

We do at least have end2end coverage right now of relevant codepaths.